cmd/root.go

```diff
@@ -31,20 +31,17 @@ func init() {
 	rootCmd.AddCommand(ConfigGetCmd)
 	rootCmd.AddCommand(ConfigSetCmd)
 	rootCmd.AddCommand(DestroyCmd)
-	rootCmd.AddCommand(InitCmd)
-	rootCmd.AddCommand(IronstarCmd)
-	rootCmd.AddCommand(UpCmd)
-	rootCmd.AddCommand(DestroyCmd)
+	rootCmd.AddCommand(ExecCmd)
+	// rootCmd.AddCommand(IronstarCmd)
 	rootCmd.AddCommand(LogsCmd)
 	rootCmd.AddCommand(OpenCmd)
 	rootCmd.AddCommand(PortsCmd)
 	rootCmd.AddCommand(PsCmd)
-	rootCmd.AddCommand(RepairCmd)
-	rootCmd.AddCommand(ExecCmd)
-	rootCmd.AddCommand(StopCmd)
-	rootCmd.AddCommand(SyscheckCmd)
 	rootCmd.AddCommand(StatusCmd)
+	rootCmd.AddCommand(StopCmd)
 	rootCmd.AddCommand(SyncCmd)
+	rootCmd.AddCommand(SyscheckCmd)
+	rootCmd.AddCommand(UpCmd)
 	rootCmd.AddCommand(UpCmd)
```

Comment:
Ooops. UpCmd was always there twice, but wasn't as obvious because the list wasn't sorted previously. I found another one like that too, but I'll fix this.